### ISSUE TITLE
Ownership of the domain flycheck.org

### ISSUE THREAD

**Issue by mgudemann** [flycheck/flycheck#1989]
Hello @all 

since 2017 I am the owner of the flycheck.org domain (see #1257). 

I am not really involved in flycheck development anymore and not at all up-to-date of what is going on in the project. Therefore I would like to pass on the baton to someone else (or a foundation etc.). The domain is registered until Nov 19th 2023 so there should be plenty of time to find a solution.

best regards

**Comment by jeremyheiler** [comment]
@mgudemann Do you still have the domain? If so, where is the site hosted?

**Comment by mgudemann** [comment]
@jeremyheiler Yes, I still have the domain. The page is a CNAME redirect to `readthedocs.io`

**Comment by jeremyheiler** [comment]
Who are the active maintainers of this project?

**Comment by mgudemann** [comment]
>  Who are the active maintainers of this project?

AFAIK @cpitclaudel 
Are you interested taking ownership of the domain?

**Comment by bbatsov** [comment]
@mgudemann I'm the new maintainer of the project, so I'd appreciate it if you can transfer the ownership of the domain to me.

**Comment by bbatsov** [comment]
(funny enough I was wondering today who happens to own the domain right now)

**Comment by mgudemann** [comment]
@bbatsov sure, happy to hear that you will take up the baton
How should we proceed? I could contact you on your mail address as on your website.

**Comment by bbatsov** [comment]
@mgudemann Yeah, that'd be best.

**Comment by mgudemann** [comment]
@bbatsov I just got the authinfo code from the hoster and sent it code to you via mail

**Comment by bbatsov** [comment]
The transfer was just completed! Thanks! 🙇
